ovs-p4rt/ovs_p4rt_session.cc

```diff
@@ -106,7 +106,7 @@ absl::Status GetForwardingPipelineConfig(OvsP4rtSession* session,
   return absl::OkStatus();
 }
 
-absl::StatusOr<ReadResponse> SendReadRequest(OvsP4rtSession* session,
+absl::Status SendReadRequest(OvsP4rtSession* session,
```

Comment:
The purpose of these changes is to simplify SendReadRequest so it returns just the status code?

You might want to add something to this effect to the commit comment, to provide good documentation for both review and audit trail purposes.